superdesk/audit/audit.py

```diff
@@ -0,0 +1,85 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2013, 2014 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+import logging
+from flask import g
+from superdesk.resource import Resource
+from superdesk.services import BaseService
+
+log = logging.getLogger(__name__)
+
+
+class AuditResource(Resource):
+    endpoint_name = 'audit'
+    resource_methods = ['GET']
+    item_methods = ['GET']
+    schema = {
+        'resource': {'type': 'string'},
+        'action': {'type': 'string'},
+        'extra': {'type': 'dict'},
+        'user': Resource.rel('users', False)
+    }
+    exclude = {endpoint_name, 'activity', 'dictionaries', 'macros'}
```

Comment:
maybe there should be also `auth` for sessions, but this might be also filtered out via `g.user`